### PR TITLE
[CLOUDTRUST-2109] PR2 authorization management

### PR DESCRIPTION
**Pull request by harture**
Uses cloudtrust/common-service#34

### CODE REVIEW

**Comment by harture** [comment]
Check all swagger for /actions

**Comment by harture** [comment]
Remove Name in messages.go
In api.go -> l:441 internal.GroupName instead of internal.GroupID

**Comment by harture** [comment]
Move ActionRepresentation into common.
TODO in all repos (i.e. also in papercard)

**Comment by harture** [comment]
@bsoniam @fperot74 Could you review it again ? I did rebase and we are now ready for merge.

File: pkg/events/authorization_test.go [hunk 1]
```diff
@@ -13,27 +14,42 @@ import (
...
-		"master": {
-			"toe": {
-				"EV_GetActions": {"*": {}},
-				"EV_GetEvents": {"*": {"*": {} }},
-				"EV_GetEventsSummary": {"*": {"*": {} }},
-				"EV_GetUserEvents": {"*": {"*": {} }}
-			}
-		}
-	}`
+const ()
```
Comment (bsoniam) [inline pkg/events/authorization_test.go]: can we  remove this?

File: api/event/fb/AdminEvent.go [hunk 2]
```diff
@@ -91,27 +91,27 @@ func (rcv *AdminEvent) DetailsLength() int {
...
 	return 0
 }
 
-func (rcv *AdminEvent) MutateResourceType(n ResourceType) bool {
+func (rcv *AdminEvent) MutateResourceType(n int8) bool {
 	return rcv._tab.MutateInt8Slot(14, n)
 }
 
-func (rcv *AdminEvent) OperationType() OperationType {
+func (rcv *AdminEvent) OperationType() int8 {
```
Comment (bsoniam) [inline api/event/fb/AdminEvent.go]: this change is related to the PR that added information on the admin event ?

File: Gopkg.toml [hunk 3]
```diff
@@ -27,11 +27,11 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/common-service"
-  version = "v1.2.4"
+  branch = "CLOUDTRUST-2109_2_authorizationManagement"
```
Comment (fperot74) [inline Gopkg.toml]: Don't forget to update before merging

File: Gopkg.toml [hunk 4]
```diff
@@ -27,11 +27,11 @@
...
 
 [[constraint]]
   name = "github.com/cloudtrust/keycloak-client"
-  version = "v1.2.7"
+  branch = "CLOUDTRUST-2109_2_branch"
```
Comment (fperot74) [inline Gopkg.toml]: Don't forget to update before merging

File: pkg/events/authorization_test.go (hunk 1 above)
Comment (harture) [reply pkg/events/authorization_test.go]: I think we can safely remove it ;)

**Pull request merged**